src/website/app/ComponentDocExample.js

```diff
@@ -16,6 +16,8 @@
 
 /* @flow */
 import React from 'react';
+import Helmet from 'react-helmet';
+import startCase from 'lodash/startCase';
```

Comment:
I don't see `lodash.startcase` in `package.json`. Are we just able to use it because of a transitive dependency?  If so, it would probably be wise to add it explicitly.

Reply:
ah, i thought we had an explicit lodash dependency. updated.